zkevm-circuits/src/evm_circuit/util/common_gadget.rs

```diff
@@ -108,6 +108,7 @@ impl<F: Field> RestoreContextGadget<F> {
         reversible_write_counter_increase: Expression<F>,
     ) -> Self {
         // Read caller's context for restore
+
```

Comment:
```suggestion
```

Reply:
removed in [faa3721](https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/1030/commits/faa372133d9079ff17be8f7346c059016dacb82a)